src/main/java/org/benetech/servicenet/conflict/detector/AbstractDetector.java

```diff
@@ -4,17 +4,27 @@
 import org.benetech.servicenet.domain.AbstractEntity;
 import org.benetech.servicenet.domain.Conflict;
 import org.benetech.servicenet.domain.enumeration.ConflictStateEnum;
+import org.benetech.servicenet.matching.counter.UrlSimilarityCounter;
+import org.springframework.beans.factory.annotation.Autowired;
 
 import java.time.ZonedDateTime;
 import java.util.LinkedList;
 import java.util.List;
 
 public abstract class AbstractDetector<T extends AbstractEntity> {
 
+    @Autowired
+    private UrlSimilarityCounter urlSimilarityCounter;
```

Comment:
you should not change the AbstractDetector, make the changes in OrganizationConflictDetector instead